packages/development/tools/misc/pester/plan.ps1

```diff
@@ -8,7 +8,11 @@ $pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 $pkg_source="https://github.com/pester/Pester/archive/$pkg_version.zip"
 $pkg_shasum="39dac4c4c19ad2ffce1e863bea8b68b05a648156833933ecde1826a49ac8f2d6"
 $pkg_bin_dirs=@("module/bin")
-
+$pkg_build_deps=@("core/dotnet-core-sdk")
+function Invoke-Build {
+	Set-Location "$pkg_name-$pkg_version"
+	.\build.ps1 -Clean
+}
 function Invoke-Install {
     Copy-Item "pester-$pkg_version\*" "$pkg_prefix/module" -Recurse -Force

```

Comment:
The entire pester repo is landing in the module directory.
I think you want this to be:
```
Copy-Item "pester-$pkg_version\bin\*" "$pkg_prefix/module" -Recurse -Force
```

Reply:
done